src/scripts/clipperUI/animations/animationStrategy.ts

```diff
@@ -1,3 +1,5 @@
+import {SmartValue} from "../../communicator/smartValue";
```

Comment:
Now that I think about, a Mith-react component tree serves a similar purpose to a SmartValue. When the state changes, it re-renders the component and passes the re-rendered value down as a prop to its children. #FoodForThought